### ISSUE TITLE
[EPIC] - Shopping Carts BO/FO

### ISSUE THREAD

**Issue by hibatallahAouadni** [PrestaShop/PrestaShop#31493]
## FO - Shopping Carts
- [x] #21533 
- [x] #10005
- [ ] #22425
- [x] #28515
- [x] #31358

### Bugs
- [ ] #13705
- [ ] #20727

### Features
- [x] #10996
- [x] #22354
- [x] #23125
- [x] #24484

#### FO - Shopping Carts
- [x] #28336
#### BO - Shopping Carts
- [x] #29894

**Comment by boo-code** [comment]
State of the three unticked items:

- **#13705** is fixed. PR #40656 (@kpodemski) merged into `9.1.x` on 2026-02-10 passes the customer's group to `SpecificPrice::getSpecificPrice()` instead of `id_shop_group`, in both `Cart::getProducts()` and `Order.php`. Details on the issue.
- **#22425** has an open PR, #38631 by @mcaldex, "fix flawed _products cache of class Cart". The earlier attempt #24825 was closed in 2021.
- **#20727** I have not checked in depth yet.

So one of the three can be ticked and closed, and one is waiting on review rather than on a fix being written.
